std_msgs/Header header
int32 xian_dj_stewart_platform_params_server_heart_beat 
int32 xian_dj_stewart_platform_control_heart_beat 
int32 xian_dj_stewart_platform_base_electric_heart_beat 
int32 xian_dj_stewart_platform_error_monitor_heart_beat 
int32 xian_dj_stewart_platform_back_end_heart_beat 
int32 xian_dj_stewart_platform_manual_controller_heart_beat 

float32 xian_dj_stewart_platform_input_alpha_cmd 
float32 xian_dj_stewart_platform_input_beta_cmd 
float32 xian_dj_stewart_platform_input_gamma_cmd 
float32 xian_dj_stewart_platform_input_x_cmd 
float32 xian_dj_stewart_platform_input_y_cmd 
float32 xian_dj_stewart_platform_input_z_cmd 

int32 xian_dj_stewart_platform_input_alpha_positive_cmd 
int32 xian_dj_stewart_platform_input_alpha_negative_cmd 
int32 xian_dj_stewart_platform_input_beta_positive_cmd 
int32 xian_dj_stewart_platform_input_beta_negative_cmd 
int32 xian_dj_stewart_platform_input_gamma_positive_cmd 
int32 xian_dj_stewart_platform_input_gamma_negative_cmd 
int32 xian_dj_stewart_platform_input_x_positive_cmd 
int32 xian_dj_stewart_platform_input_x_negative_cmd 
int32 xian_dj_stewart_platform_input_y_positive_cmd 
int32 xian_dj_stewart_platform_input_y_negative_cmd 
int32 xian_dj_stewart_platform_input_z_positive_cmd 
int32 xian_dj_stewart_platform_input_z_negative_cmd 

float32 xian_dj_stewart_platform_arm1_cmd 
float32 xian_dj_stewart_platform_arm2_cmd 
float32 xian_dj_stewart_platform_arm3_cmd 
float32 xian_dj_stewart_platform_arm4_cmd 
float32 xian_dj_stewart_platform_arm5_cmd 
float32 xian_dj_stewart_platform_arm6_cmd 

int32 xian_dj_stewart_platform_params_server_error 
int32 xian_dj_stewart_platform_control_error 
int32 xian_dj_stewart_platform_base_electric_error 
int32 xian_dj_stewart_platform_back_end_error 
int32 xian_dj_stewart_platform_error_code 